DB/postgresSeed.js

```diff
@@ -0,0 +1,106 @@
+const { Sequelize, DataTypes } = require('sequelize');
+const dataGenerator = require('./dataGenerator.js');
+const sequelize = new Sequelize('carousel', 'postgres', 'hrr48', {
+  dialect: 'postgres',
+  storage: 'localhost:5432',
+  logging: false
+})
+
+
+const test = async () => {
+  try {
+    await sequelize.authenticate();
+    console.log('Connection has been established successfully.');
+  } catch (error) {
+    console.error('Unable to connect to the database:', error);
+  }
+}
+
+
+const Product = sequelize.define('Product', {
```

Comment:
It looks like you started defining a model here, which you may want to keep in a separate, especially if you want to use sequelize as method to switch between databases quickly!

Reply:
Probably true for anything that is sequelize that is not PG specific.